src/component/main/Editor.js

```diff
@@ -0,0 +1,79 @@
+import request from "../../api.js";
+
+export default class Editor {
+
+    constructor({ pageElement, onChangeTitle }) {
+        this.onChangeTitle = onChangeTitle.bind(this);
+        const boardElement = document.createElement('div');
+        this.titleEditorElement = document.createElement('h1');
+        this.editorElement = document.createElement('form');
+
+        boardElement.className = "editor";
+        this.editorElement.className = "textEditor";
+        this.titleEditorElement.placeholder = "제목 없음";
+        this.titleEditorElement.setAttribute("contenteditable", "true");
+        this.editorElement.setAttribute("contenteditable", "true");
+
+        pageElement.appendChild(boardElement);
+        boardElement.appendChild(this.titleEditorElement);
+        boardElement.appendChild(this.editorElement);
+
+        this.setEvent();
```

Comment:
이 부분은 석현님이 하신 todo App 과제에서 드렸던 리뷰와 비슷한데요, `constructor`는 인스턴스를 초기화하는 역할을 담당합니다. 그런데, 인스턴스 초기화 이외에 돔을 생성하고 그 돔에 속성이나 하위 돔을 추가하는 등 많은 일들을 하고 있는 것 같습니다. 돔 생성 메서드를 생성하고 그 메서드를 `constructor`에서 호출하는 방식은 어떤가요? 다음과 같이요!
```suggestion
constructor({ pageElement, onChangeTitle }) {
  // 생략
  this.createDOMElements(pageElement);
  this.setEvent();
}

createDOMElements(pageElement) {
        const boardElement = document.createElement('div');
        this.titleEditorElement = document.createElement('h1');
        this.editorElement = document.createElement('form');

        boardElement.className = "editor";
        this.editorElement.className = "textEditor";
        this.titleEditorElement.placeholder = "제목 없음";
        this.titleEditorElement.setAttribute("contenteditable", "true");
        this.editorElement.setAttribute("contenteditable", "true");

        pageElement.appendChild(boardElement);
        boardElement.appendChild(this.titleEditorElement);
        boardElement.appendChild(this.editorElement);
    }
```

Reply:
오호 사실 최근들어 제가 생성자는 이것도 해도 되지않을까? 저것도 해도 되지않을까? 라는 생각에 역할을 모호하게 생각하고 있었는데, 정확한 기준을 알려주셔서 감사합니다!